network/wsNetwork.go

```diff
@@ -772,6 +778,21 @@ func (wn *WebsocketNetwork) checkServerResponseVariables(header http.Header, add
 	return true
 }
 
+// waitAndAddConnectionTime will wait to prevent exceeding connectionsRateLimitingCount.
+// Then it will register the next connection time.
+func (wn *WebsocketNetwork) waitAndAddConnectionTime(addr string) {
+	wn.phonebook.WaitAndAddConnectionTime(addr)
+}
```

Comment:
I think it's abit pointless to create a wrapper function for a single line of code..